docs/widgets.md

```diff
@@ -248,6 +248,63 @@ Note the addition required value in the entity file:
 
 The [rssToJson][] microservice is a fine way to convert desired RSS feeds into the required JSON representation.
 
+### Action Items List
+
+![action items widget](./img/action-items.png)
+
+```xml
+<name>widgetType</name>
+<value>action-items</value>
+```
+
+#### When to use
+
+* You want to display a list of quantity-based items, with quantities that are expected to change. For example, a manager who has to approve time off could see "5 leave requests need your approval".
+
+#### Additional entity file configuration
+
+```xml
+<portlet-preference>
+    <name>widgetType</name>
+    <value>action-items</value>
+</portlet-preference>
+<portlet-preference>
+    <name>widgetConfig</name>
+    <value>
+      <![CDATA[{
+        "actionItems": [
```

Comment:
That decision predates me, but I imagine it was done so the configuration was simpler to consume in the various places it's needed. @apetro can probably elaborate.